src/v0/destinations/pinterest_tag/transform.js

```diff
@@ -228,12 +232,24 @@ const processRouterDest = inputs => {
         output: transformedEvents
       };
     } catch (error) {
+      const resp = getErrorRespEvents(
```

Comment:
You need to first translate the error using `generateErrorObject`.

Reply:
updated